example/grpc_web/tavern_grpc_web_example/server.py

```
@@ -0,0 +1,61 @@
+    try:
+        req.ParseFromString(message_bytes)
+    except DecodeError:
+        return _grpc_web_response(status="3", details="invalid protobuf payload")
+
+    reply = helloworld_pb2.HelloReply(message=f"Hello, {req.name}!")
+    return _grpc_web_response(status="0", message_bytes=reply.SerializeToString())
+
+
+if __name__ == "__main__":
+    logging.basicConfig(level=logging.INFO)
+    app.run(host="0.0.0.0", port=50053)
```

Ruff (0.15.10)

[error] Possible binding to all interfaces

(S104)